packages/flutter_tools/templates/app/linux.tmpl/my_application.cc.tmpl

```diff
@@ -33,8 +35,29 @@ static void my_application_activate(GApplication* application) {
   gtk_widget_grab_focus(GTK_WIDGET(view));
 }
 
+static gboolean my_application_local_command_line(GApplication* application, gchar ***arguments, int *exit_status) {
+  MyApplication* my_app = MY_APPLICATION(application);
+  my_app->dart_entrypoint_args = *arguments;
```

Comment:
Use `g_strdupv` here - I don't think this array is guaranteed to exist outside this callback. This will require a `g_strfreev` in a `MyApplication::dispose` method.